forge-gui/res/cardsfolder/upcoming/fall_of_gil_galad.txt

```diff
@@ -0,0 +1,13 @@
+Name:Fall of Gil-galad
+ManaCost:1 G
+Types:Enchantment Saga
+K:Saga:3:DBScry,DBPutCounter,DBEffect
+SVar:DBScry:DB$ Scry | ScryNum$ 2 | SpellDescription$  Scry 2.
+SVar:DBPutCounter:DB$ PutCounter | ValidTgts$ Creature.YouCtrl | TgtPrompt$ Select target creature you control | CounterType$ P1P1 | CounterNum$ 2 | SpellDescription$ Put two +1/+1 counters on target creature.
+SVar:DBEffect:DB$ Effect | Triggers$ DiesTrig | SubAbility$ DBFight | ValidTgts$ Creature.YouCtrl | RememberTargets$ True | TgtPrompt$ Select target creature you control | SpellDescription$  Until end of turn, target creature you control gains "When this creature dies, draw two cards." Then that creature fights up to one other target creature.
```

Comment:
not a delayed trigger

Reply:
does it need to be a delayedtrigger ?

Reply:
what you scripted is a delayed trigger (via effect variant)
But the card says the creature **gains** it

Reply:
Is this via animate or via adding trigger on a static from an effect ?

Reply:
It works now but It still allows me to target the same creature for the fight ability

Reply:
Uhm..... `Other` just doesn't make sense as that's in relation to the current card

Reply:
see cards like `Arc Trail`